ephios/plugins/basesignup/templates/basesignup/section_based/fragment_state.html

```diff
@@ -2,13 +2,16 @@
 
 <div>
     {% for section in sections.values %}
-        <span><strong>{{ section.participations|length }}</strong> {{ section.title }}</span>
+        <span class="pe-1"><strong>
+            {{ section.stats.confirmed_count }}{% if section.stats.max_count %}/{{ section.stats.max_count }}{% endif %}
+        </strong> {{ section.title }}</span>
+
         <div class="mb-3">
             {% for participation in section.participations %}
                 {% include "basesignup/participation_card_inline.html" %}
             {% endfor %}
-            {% for i in section.missing %}
-                {% if section.qualifications_label %}
+            {% for i in section.placeholder %}
+                {% if False and section.qualifications_label %}
```

Comment:
???

Reply:
This was me trying out how it looks without. Will remove...